### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by rajdakin**
This PR has various fixes: some minor signed/unsigned comparisons and other cosmetic fixes, some missing returns in wrapped functions, a missing `break;` and a missing `nextop = F8;`.

This PR has been compiled on an x86_64 machine with `-Wall -Wextra -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function`, and only some false positive warnings during LTO remain.

### CODE REVIEW

File: src/wrapped/wrappedpulse.c [hunk 1]
```diff
@@ -869,11 +869,11 @@ static void* find_time_new_Fct(void* fct)
...
 }
 // time_restart
-#define GO(A)                                               \
-static uintptr_t my_time_restart_fct_##A = 0;               \
-static void* my_time_restart_##A(void* e, void* t)          \
-{                                                           \
-    RunFunctionFmt(my_time_restart_fct_##A, "pp", e, t);    \
+#define GO(A)                                                          \
+static uintptr_t my_time_restart_fct_##A = 0;                          \
+static void* my_time_restart_##A(void* e, void* t)                     \
```
Comment (ptitSeb) [inline src/wrapped/wrappedpulse.c:874]: Yeah, looks like it's `void` callback: https://freedesktop.org/software/pulseaudio/doxygen/structpa__mainloop__api.html

Comment (rajdakin) [reply src/wrapped/wrappedpulse.c:874]: Fixed.

File: src/wrapped/wrappedpulse.c [hunk 2]
```diff
@@ -891,11 +891,11 @@ static void* find_time_restart_Fct(void* fct)
     return NULL;
 }
 // time_free
-#define GO(A)                                       \
-static uintptr_t my_time_free_fct_##A = 0;          \
-static void* my_time_free_##A(void* e)              \
-{                                                   \
-    RunFunctionFmt(my_time_free_fct_##A, "p", e);   \
+#define GO(A)                                                   \
```
Comment (ptitSeb) [inline src/wrapped/wrappedpulse.c:894]: it's a `void` callback https://freedesktop.org/software/pulseaudio/doxygen/structpa__mainloop__api.html

File: src/wrapped/wrappedpulse.c [hunk 3]
```diff
@@ -957,11 +957,11 @@ static void* find_defer_new_Fct(void* fct)
     return NULL;
 }
 // defer_enable
-#define GO(A)                                               \
-static uintptr_t my_defer_enable_fct_##A = 0;               \
-static void* my_defer_enable_##A(void* e, int b)            \
-{                                                           \
-    RunFunctionFmt(my_defer_enable_fct_##A, "pi", e, b);    \
+#define GO(A)                                                          \
```
Comment (ptitSeb) [inline src/wrapped/wrappedpulse.c:960]: `void` callback

File: src/wrapped/wrappedpulse.c [hunk 4]
```diff
@@ -979,11 +979,11 @@ static void* find_defer_enable_Fct(void* fct)
     return NULL;
 }
 // defer_free
-#define GO(A)                                       \
-static uintptr_t my_defer_free_fct_##A = 0;         \
-static void* my_defer_free_##A(void* e)             \
-{                                                   \
-    RunFunctionFmt(my_defer_free_fct_##A, "p", e);  \
+#define GO(A)                                                    \
```
Comment (ptitSeb) [inline src/wrapped/wrappedpulse.c:982]: `void` calback

File: src/wrapped/wrappedpulse.c [hunk 5]
```diff
@@ -781,11 +781,11 @@ static void* find_io_new_Fct(void* fct)
     return NULL;
 }
 // io_enable
-#define GO(A)                                                   \
-static uintptr_t my_io_enable_fct_##A = 0;                      \
-static void* my_io_enable_##A(void* api, int events)            \
-{                                                               \
-    RunFunctionFmt(my_io_enable_fct_##A, "pi", api, events);    \
+#define GO(A)                                                              \
```
Comment (ptitSeb) [inline src/wrapped/wrappedpulse.c:784]: `void` callback

File: src/wrapped/wrappedpulse.c (hunk 2 above)
Comment (rajdakin) [reply src/wrapped/wrappedpulse.c]: Fixed.

**Pull request merged**